Tests/UnitTests/Core/TrackFitting/Gx2fTests.cpp

```diff
@@ -49,7 +49,7 @@ Acts::CurvilinearTrackParameters makeParameters(
     const ActsScalar x = 0.0_m, const ActsScalar y = 0.0_m,
     const ActsScalar z = 0.0_m, const ActsScalar w = 42_ns,
     const ActsScalar phi = 0_degree, const ActsScalar theta = 90_degree,
-    const ActsScalar p = 1_GeV, const ActsScalar q = 1_e) {
+    const ActsScalar p = 2_GeV, const ActsScalar q = 1_e) {
```

Comment:
Do the constant changes mean anything?

Reply:
The change is for properly testing the QoP-fit.
Now we have `QoP = 1_e / 2_GeV` as real parameters and `QoP = 1_e / 1_GeV` as start parameters. Without the magnetic field the start parameters are kept untouched. With field we get close to the real parameters.